MetaMask/metamask-extension
# feat: Support for EVM on Flow

## **Description**


Adding support for EVM on Flow

Flow blockchain supports EVM as of September 2024
this PR adds EVM on Flow to the list of additional networks so users can easily add it to their network list

[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/MetaMask/metamask-extension/pull/28825?quickstart=1)

## **Related issues**

Fixes: https://github.com/onflow/docs/issues/1006

## **Manual testing steps**

1. build the project with `yarn dist` and `yarn dist:mv2`
2. add the local build to a [chrome based browser](https://github.com/MetaMask/metamask-extension/blob/develop/docs/add-to-chrome.md) or a [firefox based browser](https://github.com/MetaMask/metamask-extension/blob/develop/docs/add-to-firefox.md)
3. open the Metamask extension
4. open switch network interface
5. there should be an `EVM on Flow Mainnet` chain in the list
6. clicking on it should successfully add it to the list of networks

## **Screenshots/Recordings**






### **Before**


![Screenshot 2024-11-29 at 16 04 26](https://github.com/user-attachments/assets/8bb7edab-9a61-4a7d-bf6e-7cbeac0943ce)


### **After**


![Screenshot 2024-11-29 at 15 53 57](https://github.com/user-attachments/assets/4e57ec32-3597-4e09-b8b6-b85fbc0ffd6a)

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
